pkg/reconciler/taskrun/resources/taskref.go

```diff
@@ -222,6 +231,26 @@ func (l *LocalTaskRefResolver) GetTask(ctx context.Context, name string) (*v1.Ta
 	return task, nil, nil, nil
 }
 
+// LocalRefResolver uses the current cluster to resolve a stepaction reference.
+type LocalRefResolver struct {
```

Comment:
just to make it specific, maybe down the road there'll be another `Ref`

```suggestion
type LocalStepActionRefResolver struct {
```